src/test/java/e2e/util/DataProviders.java

```diff
@@ -0,0 +1,30 @@
+package e2e.util;
+
+import com.github.javafaker.Faker;
+import org.testng.annotations.DataProvider;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+
+public class DataProviders {
+
+    Faker faker = new Faker();
+
+    @DataProvider
+    public Iterator<Object[]> userCanLogin(){
```

Comment:
better rename this method because it name same like method in LoginTest
name : loginValidData